.idems_app/deployments/plh/debug.config.ts

```diff
@@ -0,0 +1,15 @@
+import { cloneConfig } from "data-models/deployment.model";
+import DEFAULT_CONFIG from "./global.config";
+import { SKINS } from "./skins";
+
+/** Debug config extends the default config **/
+
+const config = cloneConfig(DEFAULT_CONFIG);
+config.name = "plh_debug";
+
+// Override constants
+config.app_config!.APP_SKINS!.defaultSkinName = SKINS.debug.name;
+// Limit available skins to only include debug skin, to force this skin to be applied on init
```

Comment:
This currently seems like the best way to ensure that the debug skin is applied when setting the active deployment to `plh_debug`. If, for example, `weekly_workshop` was an available skin for this deployment, then when switching to this deployment the `weekly_workshop` skin would remain active. However if we were to force a deployment's default skin to be applied on init, then the ParentApp would reset it's skin to `weekly_workshop` on every initialisation, overriding some users' choice of the ` modular` skin.

Reply:
I'm assuming this is a unique problem for local development, given that all other deployments sit either on their own domain or app, and so localstorage won't be cross-populated. Happy with this as a workaround